contracts/truefi2/LineOfCreditAgency.sol

```diff
@@ -694,6 +689,11 @@ contract LineOfCreditAgency is UpgradeableClaimable, ILineOfCreditAgency {
         uint8 bucketNumber,
         address borrower
     ) internal returns (uint256 totalBorrowerInterest) {
```

Comment:
Can you please make the function return explicitly?

Reply:
Good catch, but I actually like implicit return here, works as a self documentation as it's very much not clear what is returned by the function

Reply:
I just noticed newly added `return` and wanted to find second one ... aaand failed. That's how I got to it. +1 for self documentation.